CMakeLists.txt

```diff
@@ -103,8 +103,9 @@ if (IRODS_BUILD_WITH_WERROR)
   endif()
 endif()
 
-include_directories(${IRODS_EXTERNALS_FULLPATH_JSON}/include
-                    ${IRODS_EXTERNALS_FULLPATH_SPDLOG}/include)
+find_package(nlohmann_json "3.7.3" REQUIRED)
```

Comment:
just noting that `3.7.3 != 3.10.4`

Reply:
Right. We can still compile against a properly packaged 3.7.3, so I'm passing that in as the minimum version supported. We could probably go lower, but I haven't tested anything older than 3.7.3.

Reply:
gotcha - just caught my eye as different.   maybe a comment about being a 'minimum'.